.travis.yml

```diff
@@ -58,7 +58,8 @@ install:
     - sudo apt-get install cmake
 
     # Install Mongo C Driver
-    - pushd mongo-c-driver; ./autogen.sh --enable-tests=no --enable-examples=no; make; sudo make install; popd;
+    - mkdir -p /var/tmp/cdriver
+    - pushd mongo-c-driver; ./autogen.sh --enable-tests=no --enable-examples=no --prefix=/var/tmp/cdriver; make; sudo make install; popd;
```

Comment:
Is this rebased against my recent travis changes?  Wondering because `--with-libbson=bundled` isn't here.